recipes/libcurl/all/conanfile.py

```diff
@@ -331,7 +333,7 @@ def _get_configure_command_args(self):
             "--enable-tftp={}".format(yes_no(self.options.with_tftp)),
             "--enable-debug={}".format(yes_no(self.settings.build_type == "Debug")),
             "--enable-ares={}".format(yes_no(self.options.with_c_ares)),
-            "--enable-threaded-resolver={}".format(yes_no(self.options.with_c_ares)),
+            "--enable-threaded-resolver={}".format(yes_no(self.options.with_threaded_resolver)),
```

Comment:
this option has no effect in the CMake build :s

Reply:
hmmm... Yes, probably `self._cmake.definitions["ENABLE_THREADED_RESOLVER"]` should be properly set within `_configure_cmake(self)`.